lib/ProductOpener/Display.pm

```diff
@@ -1385,9 +1372,17 @@ sub get_cache_results ($key, $request_ref) {
 sub set_cache_results ($key, $results) {
 
 	$log->debug("Setting value for MongoDB query key", {key => $key}) if $log->is_debug();
+	my $result_size = total_size($results);
+
+	# memcached max object size is 1 048 576 bytes
+	if ($result_size >= 1048576) {
```

Comment:
Can we put this number as a constant in Cache.pm ? 1M is the default memcached max object size, but it could be changed.